src/js/shot-generator/DragControls.js

```diff
@@ -125,6 +126,74 @@ class DragControls extends THREE.EventDispatcher {
     }
   }
 
+  getFromSprite ( intersects ) {
+    let charsintersect = []
+    let intersect = intersects[0],
+      i = 0
+
+    while (intersects[i])
+    {
+      //selecting chars if there are more intersection results
+      if (intersects[i].object.parent.linkedTo && intersects[i].object.parent.linkedTo.userData.type === "character") {
+        intersect = intersects[i]
+        charsintersect.push(intersects[i])
+        //break
+      }
+      i++
+    }
+    if (charsintersect.length > 1)
+    {
+      intersect = this.getClosestToMouse( charsintersect )
+    }
+    if (intersect.object instanceof THREE.Sprite)
+    {
+      let obj = intersect.object.parent.linkedTo
+      return [obj, null]
+    }
+    if (intersect.object.parent instanceof THREE.Group)
+    {
+      let obj = intersect.object.parent
+      return [obj, null]
+    }
+  }
+
+  putPoint ( coord, color )
```

Comment:
could comment this out if unused

Reply:
commented out, but keeping it a bit for quick debugging